core/src/main/java/io/ddf/content/Schema.java

```diff
@@ -536,7 +536,24 @@ public static ColumnClass get(ColumnType type) {
    *
    */
   public enum DataFormat {
-    SQL, CSV, TSV, JSON
+    UNDEF, SQL, CSV, TSV, JSON, PQT;
```

Comment:
Better to use `UNDEF(-1), SQL(0), CSV(1), TSV(2), JSON(3), PQT(4)`. Also you want SQL(0) to maintain backward compat.